src/main/resources/digital/slovensko/autogram/ui/gui/about-dialog.fxml

```diff
@@ -43,6 +49,21 @@
                         <Hyperlink styleClass="autogram-body-s,autogram-link" text="GitHub-e" onAction="#githubLinkAction"/>
                         <Text styleClass="autogram-body-s" text="."/>
                     </TextFlow>
+
+                    <StackPane>
+                        <padding>
+                            <Insets top="32" bottom="16" />
+                        </padding>
+                        <children>
+                            <ImageView>
+                                <fitWidth>550</fitWidth>
+                                <preserveRatio>true</preserveRatio>
+                                <image>
+                                    <Image url="@logo OPII_ESIF.png" />
```

Comment:
Medzery v nazvoch su divne, ci to je naschval takto?

Reply:
nie je to naschvál, pridal som podtržník.